app/assets/javascripts/actions/projectBoard.js

```diff
@@ -37,6 +46,7 @@ export const fetchProjectBoard = projectId =>
       dispatch(receiveUsers(users));
       dispatch(receiveStories(stories));
       dispatch(receiveProjectBoard(projectId));
+      expandStoryIfNeeded(dispatch, UrlService.getHash);
```

Comment:
Shouldn't we follow the pattern?
Something like `dispatch(expandStoryIfNeeded(UrlService.getHash))`